editor/src/components/canvas/canvas-utils.ts

```diff
@@ -1874,57 +1873,6 @@ export function produceCanvasTransientState(
     const editorMode = editorState.mode
     switch (editorMode.type) {
       case 'insert':
-        if (
```

Comment:
I think it's probably worth throwing a big fat error here, just in case anyone tries to use this